lib/modules/datasource/terraform-module/index.ts

```diff
@@ -36,21 +41,46 @@ export class TerraformModuleDatasource extends TerraformDatasource {
       return null;
     }
 
-    const { registry, repository } =
+    const { registry: registryUrlNormalized, repository } =
```

Comment:
why rename of variable?

Reply:
because when reviewing the code, its confusing that we have both registry and registryUrl and they are actually both urls,